mono/utils/mono-error.c

```diff
@@ -437,6 +437,21 @@ mono_error_set_not_supported (MonoError *oerror, const char *msg_format, ...)
 	va_end (args);
 }
 
+
+/**
+ * mono_error_set_ambiguous_implementation:
+ *
+ * System.AmbiguousImplementationException
+ */
+void
+mono_error_set_ambiguous_implementation (MonoError *oerror, const char *msg_format, ...)
+{
+	va_list args;
+	va_start (args, msg_format);
+	mono_error_set_generic_errorv (oerror, "System.Runtime", "AmbiguousImplementationException", msg_format, args);
```

Comment:
I think you also need to update the [linker descriptor](https://github.com/mono/mono/blob/master/mcs/class/corlib/LinkerDescriptor/mscorlib.xml) to ensure that the `AmbiguousImplementationException` class is preserved.

Reply:
Pleas update the [netcore one](https://github.com/mono/mono/blob/master/mcs/class/System.Private.CoreLib/LinkerDescriptor/System.Private.CoreLib.xml) too. Thanks!

Reply:
Done! :)